Tasks/InstallAppleProvisioningProfileV1/package.json

```diff
@@ -17,9 +17,13 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.0.0",
-    "azure-pipelines-task-lib": "^2.8.0",
-    "azure-pipelines-tasks-securefiles-common": "1.0.0",
-    "azure-pipelines-tasks-ios-signing-common": "1.0.0"
+    "azure-pipelines-tasks-ios-signing-common": "2.0.0-preview",
+    "azure-pipelines-tasks-securefiles-common": "2.0.0-preview",
+    "azure-pipelines-task-lib": "^3.0.1-preview"
+  },
+  "devDependencies": {
+    "@types/mocha": "^8.0.3",
```

Comment:
For now, move these @types dependencies from `devDependencies` to `dependencies`

There is a bug in make-build.js that is causing us not to error in this case. There is work to be done to filter out `devDependencies` from the final packaged task.

Reply:
PR: #13731

Reply:
Thank you, @alex-peck I moved them.